npm/webpack-dev-server/src/helpers/nextHandler.ts

```diff
@@ -43,6 +43,7 @@ function getNextJsPackages (devServerConfig: WebpackDevServerConfig) {
     const getNextJsBaseWebpackConfigPath = require.resolve('next/dist/build/webpack-config', resolvePaths)
 
     packages.getNextJsBaseWebpackConfig = require(getNextJsBaseWebpackConfigPath).default
+    packages.nextLoadProjectInfo = require(getNextJsBaseWebpackConfigPath).loadProjectInfo
```

Comment:
```suggestion
    packages.getNextJsBaseWebpackConfig = require(getNextJsBaseWebpackConfigPath).default
    packages.nextLoadProjectInfo = require(getNextJsBaseWebpackConfigPath).loadProjectInfo
    
    const { default: getNextJsBaseWebpackConfig, loadProjectInfo } = require('getNextJsBaseWebpackConfigPath')
```

2 requires for the price of one